### ISSUE TITLE
Received Request but no action

### ISSUE THREAD

**Issue by Tuxbee** [librespot-org/librespot-java#167]
Hello,

First of all, thank you for this project.

My issue : 
- librespot-java (1.2.0) is running behind supervisor on a Raspberry (buster).
- I start playing a song from an android or apple device with ZEROCONF
- Everything works well and I can change song and so on
- If I wait for instance 30 seconds - 60 seconds and I try to change the song I received the following output (gathered from /proc/xxx/fd/1) but nothing happens !

2020-01-12 18:00:27 TRACE CdnManager:260 - Chunk 54/68 completed, cdn: audio4-ak-spotify-com.akamaized.net, cached: true, stream: {fileId: 2C79A551D41A0AA52ECEDC9A3B243647C0CC2C6A}
2020-01-12 18:00:30 TRACE CdnManager:260 - Chunk 55/68 completed, cdn: audio4-ak-spotify-com.akamaized.net, cached: true, stream: {fileId: 2C79A551D41A0AA52ECEDC9A3B243647C0CC2C6A}
**2020-01-12 18:00:30 TRACE DealerClient:76 - Received request. {mid: hm://connect-state/v1/player/command, key: b0bcaf5f-8d28-4523-b482-a24aecb31833, pid: 447950243, sender: d29babca1dd46d171befed145e8cec1f0a853c40}**
2020-01-12 18:00:33 TRACE CdnManager:260 - Chunk 56/68 completed, cdn: audio4-ak-spotify-com.akamaized.net, cached: true, stream: {fileId: 2C79A551D41A0AA52ECEDC9A3B243647C0CC2C6A}
2020-01-12 18:00:35 TRACE CdnManager:260 - Chunk 57/68 completed, cdn: audio4-ak-spotify-com.akamaized.net, cached: true, stream: {fileId: 2C79A551D41A0AA52ECEDC9A3B243647C0CC2C6A}
**2020-01-12 18:00:36 TRACE DealerClient:76 - Received request. {mid: hm://connect-state/v1/player/command, key: 849f5ac3-4709-4233-a720-d21516089e12, pid: 447955562, sender: d29babca1dd46d171befed145e8cec1f0a853c40}**

- If I play the song on the phone it works (both songs are played)
- If I try to reconnect, I receive a message "client already connected"

I know that my Wifi (used by the phones) isn't stable but the raspberry is wired connected.
Do you have any idea why it is happening ?
It is maybe a performance issue on the raspberry, do you think I need to disable the cache ?

Regards.

Louis

P.S. : I can try the dev branch if you need it !

**Comment by devgianlu** [comment]
The fact that `Received request.` is not followed by `Handled request.` probably means that the client is taking an exceptional amount of time to process the request or a deadlock occurred.

I cannot reproduce the issue, but I've just pushed a commit (26c1ebb40045024dd58081f66478bb0071ff1d54) that adds a little bit of logging to hopefully get a better insight on what's happening. You can post the log here after reproducing it. 

If this doesn't work I'll need to you to create a dump, but don't worry about it now.

**Comment by Tuxbee** [comment]
Hello,

The problem can come from my configuration.
By using the dev branch, the problem occured quickly and I was unable to listen even the first seconds of the song.
Then I outputed the stdout to investigage the problem and the song began to run properly...
Thus, I think the problem can come from the stdout redirect done by supervisord.
I only added two configuration settings to the supervised program : 
stdout_logfile=NONE
stderr_logfile=NONE
It seemed to solve the problem for the dev branch, I retried the v1.2.0 release and the problem seems also solved for the current release.
I can't test a long run, I hope that there isn't two problems in a row, a deadlock and the fact that with all the outputs the supervisord redirect fails...
I let the ticket open for now, if the problem occurs again, I will investigate more.

Regards.

Louis
